cypress/support/commands.js

```diff
@@ -216,3 +216,19 @@ Cypress.Commands.add('createResponseTemplate', ({ title, content }) => {
     `utf8=%E2%9C%93&response_template%5Btitle%5D=${encodedTitle}&response_template%5Bcontent%5D=${encodedContent}`,
   );
 });
+
+/**
+ * Helper to get a form element by its label
+ *
+ * @param {string} label The form element's label.
+ *
+ * @returns {Cypress.Chainable<Element>} The form input element.
+ */
+Cypress.Commands.add('getInputByLabel', (label) => {
```

Comment:
@nickytonline @aitchiss When writing e2e tests for my `SiteConfig` work I often wish we had this, so I decided to finally add it. I guess I'll eventually go back and simplify some of the tests I wrote previously.

Reply:
Cypress actually has a command for this already (via testing-library)! It's `cy.findByLabelText('labelText')`

Reply:
Ugh, I didn't know. The one I added here was actually taken from the Cypress documentation. I guess I'll remove this and use `findByLabelText` instead, thanks!